src/utils/api.js

```diff
@@ -1,21 +1,31 @@
 import axios from "axios";
 
 const articleApi = axios.create({
-    baseURL:'https://nc-positive-news.herokuapp.com/api'
-})
+  baseURL: "https://nc-positive-news.herokuapp.com/api",
+});
 
-export const getArticles = (slug) =>{
-    if(slug){
-    return articleApi.get('/articles',{params: {topic :`${slug}`}}).then(({data})=>{
-        return data.articles
-    })} else {return articleApi.get('/articles').then(({data})=>{
-        return data.articles
+export const getArticles = (slug) => {
+  if (slug) {
+    return articleApi
+      .get("/articles", { params: { topic: `${slug}` } })
+      .then(({ data }) => {
+        return data.articles;
+      });
+  } else {
```

Comment:
We don't actually need to conditional statement at all here!

Axios is clever enough that it will only attach queries if the params are **not** `undefined`

Reply:
You could refactor to just have a single get with params and axios would take care of the rest :)